docs/git-comparison.md

```diff
@@ -183,6 +183,11 @@ parent.
       <td><code>jj log</code></td>
       <td><code>git log --oneline --graph --decorate</code></td>
     </tr>
+    <tr>
+      <td>Show log of ancestors of the current commit only</td>
```

Comment:
Should we also point out that `jj log -r ::@` doesn't just *shrink* the set to be ancestors of the current commit but also *expands* it to include *all* ancestors of it rather than stopping at immutable commits?

Reply:
If I understand it correctly `git log` also shows the entire history.

Reply:
Right, but `jj log` does not. And I've heard people wonder where all the ancestor commits went. So maybe it would be helpful to also explain that here?

Reply:
I'm not sure I understand this behavior well enough to explain it but maybe this is what you mean:

> Show log of ancestors of the current commit only, expanding to include all ancestors, not just stopping at immutable commits.

Reply:
I'd change "expanding to include" to "including" but otherwise that sounds good to me, thanks.